### PR TITLE
bpo-32429: Abort compilation on outdated Modules/Setup file.

### PR DESCRIPTION
**Pull request by JulienPalard**
https://bugs.python.org/issue32429

### CODE REVIEW

**Review by pitrou — APPROVED** [review]

File: Makefile.pre.in [hunk 1]
```diff
@@ -692,7 +692,10 @@ Modules/Setup: $(srcdir)/Modules/Setup.dist
 		echo "check to make sure you have all the updates you"; \
 		echo "need in your Modules/Setup file."; \
 		echo "Usually, copying Modules/Setup.dist to Modules/Setup will work."; \
+		echo "Or if you want to keep your modifications, touch Modules/Setup"; \
```
Comment (xdegaye) [inline Makefile.pre.in:695]: When Modules/Setup.dist has been changed as for example by 7d895ac953d2ba8c6cb9ade995aef131ddc77ab2 in September 2016, touching Modules/Setup is not sufficient. I suggest to remove that line.

**Review by xdegaye — APPROVED** [review]
LGTM, see my comment

**Comment by taleinat** [comment]
Given that PR #8229 (bpo-32430) has been merged, ISTM this should be closed.

**Comment by JulienPalard** [comment]
That's right.

**Pull request closed without merging**